apps/client/src/composables/aas-editor.spec.ts

```
@@ -81,10 +83,48 @@ vi.mock("../lib/api-client", () => ({
+    return {
+      wrapper,
+      ...wrapper.vm.api as ReturnType<typeof useAasEditor>,
+    };
+  }
+
   beforeEach(() => {
     vi.resetAllMocks();
   });
+
+  afterEach(() => {
+    mountedWrappers.splice(0).forEach(w => w.unmount());
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)